### DISCUSSION TITLE
Reverse Proxy Setup

### DISCUSSION THREAD

**Discussion by laserjet25** [NamelessMC/Nameless#3419] [Q&A] [answered]
I have been trying to setup NamelessMC behind a reverse proxy (NGINX host & NGINX proxy) but have not been able to find any documentation relating to this. I noticed that the API shows http when the proxy adds HTTPS and am getting 404s from the proxy.

**Comment by Derkades** [comment, marked as answer]
Make sure to add proper proxy headers (X-Forwarded-For, X-Forwarded-Proto, X-Forwarded-For) so Nameless is aware of the client IP address and protocol used by the proxy.

**Comment by laserjet25** [comment]
I ended up finding [this](https://github.com/NamelessMC/Nameless/issues/2018) yesterday a few hours after posting the question. After setting X-Forwarded-For on my proxy, Nameless started working as expected.

Would definitely be helpful if this could be added in the webserver config wiki page: https://docs.namelessmc.com/en/webserver
